packages/utils/src/utils.js

```diff
@@ -31,8 +49,9 @@ export const getV3BrowserResponse = async () =>
     region: REGION,
     credentials: fromCognitoIdentityPool({
       client: new CognitoIdentityClient({
-        region: REGION,
+        region: "us-east-2",
       }),
       identityPoolId: IDENTITY_POOL_ID,
     }),
+    systemClockOffset: -3600000,
```

Comment:
The `systemClockOffset` configuration needs to be passed to the client, and not `fromCognitoIdentityPool` function. In this repro, the line 56 needs to be moved to below line 49.

Reply:
I believe it already is outside of the `fromCognitoIdentityPool` object. I can certainly move it though to make it more clear